### PR TITLE
[DBZ-1837] Setting repository enforcer rule to WARN in CI tests

### PR DESCRIPTION
**Pull request by jcechace**

### CODE REVIEW

**Comment by jpechane** [comment]
@jcechace Please add prefix `[jenkins-jobs]` to the commit message

**Comment by jcechace** [comment]
I also have a JIra for this which I forgot about DBZ-1837

**Comment by jcechace** [comment]
@jpechane should be good to merge

File: jenkins-jobs/job-dsl/mongodb_matrix_test.groovy [hunk 1]
```diff
@@ -61,6 +61,7 @@ fi
 mvn clean install -U -s $HOME/.m2/settings-snapshots.xml -pl debezium-connector-mongodb -am -fae \
     -Dmaven.test.failure.ignore=true \
     -Dversion.mongo.server=$MONGODB_VERSION \
+    -Dinsecure.repositories=WARN \
```
Comment (gunnarmorling) [inline jenkins-jobs/job-dsl/mongodb_matrix_test.groovy]: @jcechace, couldn't we also just use secure repos?

**Pull request merged**